tests/unit/test_clients/test_cosmos_ledger.py

```diff
@@ -1,575 +0,0 @@
-#
-#     def test_get_balance(self):
-#         """Test get balance for the positive result."""
-#
-#         content = {"balance": {"denom": "stake", "amount": "1234"}}
-#
-#         mock_rest_client = MockRestClient(json.dumps(content))
-#         self.ledger.bank_client = BankRestClient(mock_rest_client)
-#         response = self.ledger.get_balance("address", "stake")
-#
-#         assert response == 1234
-#         assert (
-#             mock_rest_client.last_base_url == "/cosmos/bank/v1beta1/balances/address/"
-#         )
-#
-#     def test_get_balances(self):
-#         """Test get balances for the positive result."""
-#
-#         content = {
-#             "balances": [{"denom": "stake", "amount": "1234"}],
-#             "pagination": {"next_key": None, "total": 0},
-#         }
-#
-#         mock_rest_client = MockRestClient(json.dumps(content))
-#         self.ledger.bank_client = BankRestClient(mock_rest_client)
-#         response = self.ledger.get_balances("address")
-#
-#         assert str(response) == '[denom: "stake"\namount: "1234"\n]'
-#         assert mock_rest_client.last_base_url == "/cosmos/bank/v1beta1/balances/address"
-#
-#     def test_query_account_data(self):
-#         """Test query account data for the positive result."""
-#
-#         content = {
-#             "account": {
-#                 "@type": "/cosmos.auth.v1beta1.BaseAccount",
-#                 "address": "fetch1h6974x4dspft29r9gyegtajyzaht2cdh0rt93w",
-#                 "pub_key": {
-#                     "@type": "/cosmos.crypto.secp256k1.PubKey",
-#                     "key": "A2BjpEo54gBpulf9CrA+6tGBASFC8okaO1DYTimk/Jwp",
-#                 },
-#                 "account_number": "0",
-#                 "sequence": "1",
-#             }
-#         }
-#         account_response = ParseDict(content, QueryAccountResponse())
-#
-#         account = BaseAccount()
-#         if account_response.account.Is(BaseAccount.DESCRIPTOR):
-#             account_response.account.Unpack(account)
-#
-#         mock_rest_client = MockRestClient(json.dumps(content))
-#         self.ledger.auth_client = AuthRestClient(mock_rest_client)
-#         response = self.ledger.query_account_data("address")
-#
-#         assert response == account
-#         assert mock_rest_client.last_base_url == "/cosmos/auth/v1beta1/accounts/address"
-#
-#     def test_query_contract_state(self):
-#         """Test query contract state for the positive result."""
-#
-#         raw_content = b'{"data": {"balance":"1"}}'
-#         expected_response = {"balance": "1"}
-#
-#         mock_rest_client = MockRestClient(raw_content)
-#         self.ledger.wasm_client = CosmWasmRestClient(mock_rest_client)
-#         response = self.ledger.query_contract_state("fetchcontractaddress", {})
-#
-#         assert response == expected_response
-#         assert (
-#             mock_rest_client.last_base_url
-#             == "/wasm/v1/contract/fetchcontractaddress/smart/e30="
-#         )
```

Comment:
Why delete?

Reply:
all the code is commented.
Either I need to add files in ignore files or delete it.

Reply:
flake8-eradicate will raise the error for every single line.

Reply:
I added it.